easybuild/easyconfigs/f/foss/foss-2015.01.eb

```diff
@@ -0,0 +1,39 @@
+easyblock = "Toolchain"
+
+name = 'foss'
+version = '2015.01'
+
+homepage = '(none)'
+description = """GNU Compiler Collection (GCC) based compiler toolchain, including
+ OpenMPI for MPI support, OpenBLAS (BLAS and LAPACK support), FFTW and ScaLAPACK."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+comp_name = 'GCC'
+comp_version = '4.9.2'
+comp = (comp_name, comp_version)
+
+blaslib = 'OpenBLAS'
+blasver = '0.2.13'
+blas = '%s-%s' % (blaslib, blasver)
+blassuff = '-LAPACK-3.5.0'
+
+# toolchain used to build foss dependencies
+comp_mpi_tc_name = 'gompi'
+comp_mpi_tc_ver = "%s" % version
+comp_mpi_tc = (comp_mpi_tc_name, comp_mpi_tc_ver)
+
+# compiler toolchain depencies
+# we need GCC and OpenMPI as explicit dependencies instead of gompi toolchain
+# because of toolchain preperation functions
+dependencies = [
+    comp,
+    ('zlib', '1.2.8', '', comp),
+    ('binutils', '2.25', '', comp),
```

Comment:
Well, we're never gonna isolate GCC completely from the system. There is still glibc.

We could make a `GCC-4.9.2-binutils-2.25-zlib-1.2.8.eb` like thing. 

@boegel any thoughts?